### ISSUE TITLE
Shorten "Onion Available" button text to "Onion"

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#21028]
## Description 

Shorten "Onion Available" button text to "Onion"

## Steps to Reproduce

   1. new profile
   2. click on the 3-dots menu and choose `New Private Window with Tor`
   3. load `brave.com` in that window
   4. look at the `Onion Available` button in the URL bar

## Actual result:

<img width="1627" alt="Screen Shot 2022-02-10 at 1 17 52 PM" src="https://user-images.githubusercontent.com/387249/153499281-e84e91b0-3721-4782-944e-fcbae1cae599.png">

## Expected result:

Maybe we can shorten this to just `Onion`, as we've done for `Tor` and `IPFS` (see below)

For background on the two other buttons:
* https://github.com/brave/brave-browser/issues/19591
* https://github.com/brave/brave-browser/issues/18923

## Reproduces how often: 

100%

## Brave version (brave://version info)

Brave | 1.37.40 Chromium: 98.0.4758.87 (Official Build) nightly (x86_64)
-- | --
Revision | e4cd00f135fb4d8edc64c8aa6ecbe7cc79ebb3b2-refs/branch-heads/4758@{#1002}
OS | macOS Version 11.6.1 (Build 20G224)

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

cc @nullhook @diracdeltas @rebron

**Comment by GeetaSarvadnya** [comment]
Issue is fixed by the PR https://github.com/brave/brave-core/pull/12261 and can be closed as per https://github.com/brave/brave-browser/issues/21092#issuecomment-1041347682

**Comment by GeetaSarvadnya** [comment]
Verification PASSED on

```

Brave | 1.37.52 Chromium: 98.0.4758.102 (Official Build) nightly (64-bit)
-- | --
Revision | 273bf7ac8c909cde36982d27f66f3c70846a3718-refs/branch-heads/4758@{#1151}
OS | Windows 10 Version 21H2 (Build 19044.1526)

```
- Verified the STR from the description and ensured `Onion Avaialbel` text is shortened to `onion`
![image](https://user-images.githubusercontent.com/38657976/154266768-69990815-7976-407f-8ed9-87923473476b.png)
